implants/eldritch/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-starlark = { git = "https://github.com/hulto/starlark-rust", branch = "v0.6.0-hulto" }
+starlark = { git = "https://github.com/facebookexperimental/starlark-rust", rev = "7c6986a189c45591da9346159f5ddd50685bbf96" }
```

Comment:
Pinned to a commit since no release is available for 0.9.0 yet.
0.9.0 is the only official version that contains the patches for rust struct size mismatch.